src/styles/blocks/catalog.scss

```
@@ -21,5 +21,13 @@
     &:not(:nth-child(4n)) {
       margin-right: 48px;
     }
+
+    transition-duration: 300ms;
+    transition-property: color, transform;
```

you can also write this in one line
```suggestion
    transition: transform 300ms color 300ms;
```